src/components/ConfirmTermination/ConfirmTermination.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { ConfirmDialog } from '@amsterdam/bmi-component-library';
+import { BackDropStyle } from './ConfirmTerminationStyles';
+import { createPortal } from 'react-dom';
+
+export interface IState {
```

Comment:
Why do you need to declare this? Shouldn't this be dictated from the @amsterdam/bmi-component-library end?

Reply:
As discussed in Slack: IState was needed in the test. I've removed it from the component and implemented a solution that you suggested. TypeScript was still complaining for some reason so I still  needed to introduce a local (albeit minimal version of) IState interface in the test file. Maybe you could take a look to see if I did something wrong?